common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4926,9 +4926,14 @@
           "Catalog <catalogName> does not support <operation>."
         ]
       },
+      "CLAUSE_WITH_PIPE_OPERATORS" : {
```

Comment:
Maybe @srielau can also comment here, but it looks to me that we have two new errors:
1. WINDOW in the SQL pipe operator. We don't support window definition in SQL pipe yet.
2. More than one QUERY_RESULT_CLAUSES in SQL pipe. SQL pipe is designed to specify one operator at a time, so we don't allow any combination, such as ORDER BY col LIMIT 1, or LIMIT 1 OFFSET 1

Reply:
1. appears to be a 0A000 (unsupported feature)
2. Isn't that a syntax error?

Reply:
yes, 2 is a syntax error for the pipe statement.

Reply:
Thanks, split this into two new errors per suggestion.